project/plugins.sbt

```
@@ -1,7 +1,7 @@
 addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.3.1")
 
 addSbtPlugin("de.heikoseeberger" % "sbt-header" % "5.2.0")
-addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.0.0-RC5")
+addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.0.6")
```

```suggestion
addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.3.0")
```
on this version I didn't see 'search state exploded'

---

good, updated